extension/src/popup/helpers/useNetworkFees.ts

```diff
@@ -15,7 +15,7 @@ export const useNetworkFees = () => {
   const { networkUrl, networkPassphrase } = useSelector(
     settingsNetworkDetailsSelector,
   );
-  const [recommendedFee, setRecommendedFee] = useState("100");
+  const [recommendedFee, setRecommendedFee] = useState("");
```

Comment:
the bad merge that introduced this: https://github.com/stellar/freighter/pull/1398/files#diff-48e5eb06e07934afbdf9c6be1ad5d87bb1658ee0c72641914788d5655df34d2dR18